### PR TITLE
Add Training and CPD section to job application

### PR DESCRIPTION
**Pull request by KyleMacPherson**
## Trello card URL

https://trello.com/c/pWFL7Dx1/918-add-cpd-section-application

## Changes in this PR:

This PR adds a Training and CPD section to the application form which allows jobseekers to enter details about training and continuing professional development courses they have completed and displays this data to the hiring staff.

## Next steps:

- [ ] Terraform deployment required?

- [ ] New development configuration to be shared?

### CODE REVIEW

File: app/models/jobseeker_profile.rb [hunk 1]
```diff
@@ -70,6 +70,13 @@ def replace_employments!(new_employments)
     end
   end
 
+  def replace_training_and_cpds(new_training_and_cpds)
```
Comment (scruti) [inline app/models/jobseeker_profile.rb:73]:
```suggestion
  def replace_training_and_cpds!(new_training_and_cpds)
```

**Review by scruti — APPROVED** [review]
Approved with a question/name convention suggestion!

File: app/models/jobseeker_profile.rb (hunk 1 above)
Comment (KyleMacPherson) [reply app/models/jobseeker_profile.rb:73]: Ah yeah, good call. Will make that change!

Yeah it's used here! https://github.com/DFE-Digital/teaching-vacancies/pull/6835/files#diff-fa61ca8687a1914d46866fd3c81eddf807b27cb0076442bdf1b96d03100966f7R109

Comment (scruti) [reply app/models/jobseeker_profile.rb]: Thanks for checking!

**Pull request merged**